app/src/main/res/values/styles.xml

```diff
@@ -34,4 +34,70 @@
 
     <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Light" />
 
+    <style name="ScreenHeader">
+        <item name="android:textColor">@android:color/black</item>
+        <item name="android:layout_marginLeft">@dimen/fab_margin</item>
+        <item name="android:layout_marginTop">@dimen/fab_margin</item>
```

Comment:
these seem to be used very often. Maybe make another style for it?

Reply:
A lot of my styles inherit from others. I can't inherit from multiple styles so it's probably not the case.